prow/pj-on-kind.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Requires go, docker, and kubectl.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Prep and check args.
+job="${1:-""}"
+config="${CONFIG_PATH:-""}"
+job_config="${JOB_CONFIG_PATH:-""}"
+out_dir="${OUT_DIR:-/tmp/prowjob-out}"
```

Comment:
nit: if you're going to quote the empty strings for defaults above you might as well quote  this default, too :)